packages/kotti-ui/source/kotti-navbar/types.ts

```diff
@@ -17,14 +17,13 @@ export namespace KottiNavbar {
 	})
 	export type QuickLink = z.infer<typeof quickLinkSchema>
 
-	export const sectionLinkSchema = z
-		.object({
-			icon: yocoIconSchema,
-			isActive: z.boolean(),
-			link: z.string().optional(),
-			title: z.string(),
-		})
-		.passthrough()
+	export const sectionLinkSchema = z.object({
+		icon: yocoIconSchema,
+		component: z.string().default('a'),
```

Comment:
sort `component` above `icon`

Reply:
Idk where my mind is. I thought I left a comment but i didn't apparently 😕 

I had a comment about `default` here

I see you default the type, but you don\t actually `default` it on `VueProps` and you had a runtime fallback on the template itself

`:is=link.component?link.component:'a'`
which should just be purely default `a`

Reply:
nvm found it